lib/mastodon/emoji_cli.rb

```diff
@@ -96,5 +152,13 @@ def color(green, _yellow, red)
         :red
       end
     end
+
+    def export_file_name(emoji)
+      if options[:shortcode]
+        emoji.shortcode + '.png'
```

Comment:
I am not sure but do emojis need to be a png file?

Reply:
It looks to me like they are converted to png in app/models/custom_emoji.rb:36